src/web/app/components/question-responses/gqr-rqg-view-responses/gqr-rqg-view-responses.component.html

```diff
@@ -48,6 +53,11 @@
       <div class="card-header bg-primary text-white" (click)="userInfo.value.isExpanded = !userInfo.value.isExpanded">
         {{ isGqr ? 'From' : 'To' }}: {{ userInfo.key }}
         <div class="float-right">
+          <a routerLink="/web/student/sessions/submission" *ngIf="isGqr && !(userInfo.key | isIdentifierAnonymous)"
```

Comment:
You can try to extract this chunk of code to a component (maybe called as `ModerationButton`) and put all checks inside the component.